ui/index.d.ts

```diff
@@ -620,3 +615,18 @@ export declare namespace DialogResponses {
     export const dontUpload: MessageItem;
     export const reportAnIssue: MessageItem;
 }
+
+/**
+ * Call this to register common variables used throughout the UI package.
+ */
+export declare function registerUIExtensionVariables(extVars: UIExtensionVariables): void;
+
+/**
+ * Interface for common extension variables used throughout the UI package.
+ */
+export interface UIExtensionVariables {
+    context: ExtensionContext;
+    outputChannel: OutputChannel;
+    ui: IAzureUserInput;
```

Comment:
Tests will need to change the IAzureUserInput, right?  Or do they refill it with data?

Reply:
Oh, you're not making a copy, you're keeping a reference...

Reply:
Yep it's easy for tests to change it since it's a reference